packages/sdk/src/sway/predicates/predicateFinder.ts

```diff
@@ -0,0 +1,41 @@
+import { decompressBytecode } from 'fuels';
+
+export function getLatestPredicateVersion() {
+  // load versions
+  const versions = require('./versions.json');
+
+  // get latest version by time
+  const keys = Object.keys(versions);
+  let maxTime = -Infinity;
+  keys.forEach((key) => {
+    const currentTime = versions[key].time;
+    if (currentTime > maxTime) {
+      maxTime = currentTime;
+    }
+  });
+
+  // get latest version object
+  const key = keys.find((key) => versions[key].time === maxTime) ?? keys[0];
+
+  return {
+    bytecode: versions[key].bytecode,
+    abi: versions[key].abi,
+  };
+}
+
+export function loadPredicate(version?: string) {
```

Comment:
Just a question to understand better: 
Does the version come from the server-side?